.github/workflows/sync-pot.yml

```diff
@@ -0,0 +1,39 @@
+name: sync-pot
+on:
+  schedule:
+    # Once a week, on Tuesday evening
+    - cron: '0 18 * * 2'
```

Comment:
I think in this direction it doesn't hurt to run it every day -- the earlier translators have up to date POT files, the better. Conceptually, it would even be good to do this whenever we land a PR, but of course for that we'd have to run that from cockpit. Daily feels like "good enough".

Reply:
Agreed. Updating it daily seems fine. We don't want to do it too often, as Weblate itself needs to rebase, but once a day should be fine and should give us better translations for releases (as we are always at least a week behind with translations when we release)